build.sbt

```diff
@@ -113,6 +113,8 @@ lazy val core = projectMatrix
       "smithy.waiters",
       "smithy4s.api"
     ),
+    genDiscoverModels := true,
+    Test / genDiscoverModels := true,
```

Comment:
I had to set that to `true` to make the build pass, but I'm unsure because you had set it to false explictly. So keep an eye on it @Baccata